### PR TITLE
Update ResNet README for multi worker.

### PR DESCRIPTION
**Pull request by dubey**

### CODE REVIEW

File: official/resnet/README.md [hunk 1]
```diff
@@ -80,7 +80,22 @@ The appropriate distribution strategy is chosen based on the `--num_gpus` flag.
 By default this flag is one if TensorFlow is compiled with CUDA, and zero
 otherwise.
 
-num_gpus:
+It is also possible to run with multiple workers using the experimental
```
Comment (guptapriya) [inline official/resnet/README.md]: Is this sufficient? Or does the user need to specify the distribution_strategy flag too?

File: official/resnet/README.md [hunk 2]
```diff
@@ -80,7 +80,22 @@ The appropriate distribution strategy is chosen based on the `--num_gpus` flag.
...
+with the help of `--worker_hosts` and `--task_index` flags.  Use
+`--worker_hosts` to specify the list of IP:port pairs for all workers in the
+cluster, e.g.  `--worker_hosts=ip0:port0,ip1:port1`.  For each worker, specify
+the index of the worker, e.g. `--task_index=0`.  The `worker_hosts` value will
+be identical at all workers; `task_index` will be (0, 1, ... , num_workers - 1)
+respectively at each worker.  Additionally, choose the collective op
+implementation for `MultiWorkerMirroredStrategy` via the `--all_reduce_alg`
+flag; possible choices include `ring` (based on gRPC) and `nccl` (based on
+NVIDIA NCCL).
+
```
Comment (guptapriya) [inline official/resnet/README.md]: Can we add a sample command to make this more obvious?

File: official/resnet/README.md [hunk 3]
```diff
@@ -80,7 +80,22 @@ The appropriate distribution strategy is chosen based on the `--num_gpus` flag.
 By default this flag is one if TensorFlow is compiled with CUDA, and zero
```
Comment (guptapriya) [inline official/resnet/README.md]: You should feel free to re-organize this whole section to fit your needs. the current organization seems a bit odd to me, and we don't necessarily need to shoe-horn into it.

**Comment by jaeyounkim** [comment]
Deprecated

**Pull request closed without merging**